bundles/org.eclipse.orion.client.ui/web/orion/banner/banner.html

```diff
@@ -1,7 +1,5 @@
-<header id="banner" role="banner" class="headerLayout">
-	<nav id='navbar'></nav>
-	<div id="notificationArea" class="notificationHide">
-		<div class="notifications" id="notifications" aria-live="assertive" aria-atomic="true"></div>
-		<div class="layoutRight"><button type="button" aria-label="Close" class="dismissButton layoutRight core-sprite-close imageSprite" id="closeNotifications"></span></div>
-	</div>
-</header>
+<div id="banner" class="headerLayout"></div>
```

Comment:
Have you tried the WebIDE with these changes? I think this will break the notification area.

Reply:
I did test in Web IDE, but for some reason I added "hidden". I've removed "hidden" and tested again, and  notifications work.